README.md

```diff
@@ -1,2 +1,110 @@
-# many
-Rust implementation of the MANY protocol.
+# many-rs
+![ci](https://img.shields.io/github/workflow/status/liftedinit/many-rs/CI)
+![license](https://img.shields.io/github/license/liftedinit/many-rs)
+
+Rust libraries for the [MANY protocol](https://github.com/many-protocol).
+
+Features
+- MANY module interfaces
+- MANY common types
+- MANY message and transport layers
+- MANY client and server
+- Hardware Security Module
+- CLI developer's tools
+
+# References
+
+- Concise Binary Object Representation (CBOR): [RFC 8949](https://www.rfc-editor.org/rfc/rfc8949.html)
+- CBOR Object Signing and Encryption (COSE): [RFC 8152](https://datatracker.ietf.org/doc/html/rfc8152)
+- Platform-independent API to cryptographic tokens: [PKCS #11](https://docs.oasis-open.org/pkcs11/pkcs11-base/v2.40/os/pkcs11-base-v2.40-os.html)
+
+# Developer tools
+- CBOR playground: [CBOR.me](https://cbor.me)
+- CBOR diagnostic utilities: [cbor-diag](https://github.com/cabo/cbor-diag)
+- Software Hardware Security Module (HSM): [SoftHSM2](https://github.com/opendnssec/SoftHSMv2)
+
+# Installation
+
+1. Update your package database
+```shell
+# Ubuntu
+$ sudo apt update
+
+# CentOS
+$ sudo yum update
+
+# Archlinux
+$ sudo pacman -Syu
+
+# macOS
+$ brew update
+```
+2. Install Rust using [rustup](https://rustup.rs/)
+```shell
+# Ubuntu/CentOS
```

Comment:
You can just use those instructions for all platforms, no?

Reply:
Yes, it should work on all (unix) platforms.